common/pom.xml

```diff
@@ -54,6 +54,16 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>3.0</version>
 		</dependency>
+		<dependency>
```

Comment:
No idea how this keeps getting added to the pom. I'll remove before I merge into develop